packages/validate/test/StringValidators.test.js

```diff
@@ -111,4 +112,16 @@ describe('String Validation', () => {
     isEnabled = validator.execute('foo@120.120.120.93');
     expect(isEnabled).to.be.true;
   });
+
+  it('provides new Pattern() to allow only valid patterns', () => {
+    let isEnabled;
+    const validator = new Pattern(/#LionRocks/);
+    expect(validator.constructor.name).to.equal('Pattern');
```

Comment:
We should test for `validator.constructor.validatorName`.
(We probably do this wrong in the other tests as well (we couldn't use `constructor.name` because it gets lost in code minification/transpilation) )